src/pspm_sf_dcm.m

```diff
@@ -18,19 +18,23 @@
 %       .yhat:  fitted time series
 %      .model:  information about the DCM inversion
 % ● Arguments
-%         scr:  skin conductance epoch (maximum size depends on computing
-%               power, a sensible size is 60 s at 10 Hz)
-%          sr:  sampling rate in Hz
-%     options:  options structure
-%  .threshold:  threshold for SN detection (default 0.1 mcS)
-%      .theta:  a (1 x 5) vector of theta values for f_SF
-%               (default: read from pspm_sf_theta)
-%      .fresp:  frequency of responses to model (default 0.5 Hz)
-%    .dispwin:  display progress window (default 1)
-%   .dispsmallwin:
-%               display intermediate windows (default 0);
-%    .missing:  index of missing values to ignore
-%   .missingthresh:
+%   ┌──────model
+%   │ ▶︎ Mandatory
+%   ├───────.scr:  skin conductance epoch (maximum size depends on computing
+%   │              power, a sensible size is 60 s at 10 Hz)
+%   ├────────.sr:  sampling rate in Hz
+%   │ ▶︎ Optional
+%   └.missing_data: the datafile of missing epochs
+%
+%   ┌────options:  options structure
+%   ├─.threshold:  threshold for SN detection (default 0.1 mcS)
+%   ├─────.theta:  a (1 x 5) vector of theta values for f_SF
+%   │              (default: read from pspm_sf_theta)
+%   ├─────.fresp:  frequency of responses to model (default 0.5 Hz)
+%   ├───.dispwin:  display progress window (default 1)
+%   ├.dispsmallwin:
+%   │              display intermediate windows (default 0);
+%   └─.missingthresh:
```

Comment:
this option must be defined via  pspm_sf but it is not included there?

Reply:
`.missingthresh` is now used in `pspm_sf_dcm` at line 158 to check if the missing epochs are too long... Currently, `model.missing` (missing epoch file) is calculated into `model.missing_data` (missing epoch index, n x 1) by `pspm_sf` and the latter is transferred to `pspm_sf_dcm`. `model.missing_data` is then calculated into `missing_epoch` (missing epoch start and end value matrix, m x 2, where m is the number of start / end values of missing epochs) by `pspm_sf_dcm`. `.missingthresh` is then used to evaluate the `missing_epoch` (line 158 in `pspm_sf_dcm`) to check if the missing epochs are too long.
I think, if necessary, we can add `.missingthresh` to `pspm_sf` and let `pspm_sf_dcm` to inherit it if that is more intuitive. Currently `pspm_sf` is not using `.missingthresh` because `pspm_sf` does not include the step of converting `.missing_data` into `missing_epoch`.

Reply:
I understand, but the user is going to look at the help for pspm_sf first, so the option should be mentioned there.

Reply:
Sure. The option field `.missingthresh` is now available in `pspm_sf` by `pspm_options` and will be inherited by `pspm_sf_dcm` when it is called.